assets/src/design-system/icons/checklist.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M22.5 7.00018h-9v2h9v-2zm0 8.00002h-9v2h9v-2zm-16.46-4L2.5 7.46018l1.41-1.41 2.12 2.12 4.24-4.24 1.41 1.41-5.64 5.66002zm0 8l-3.54-3.54 1.41-1.41 2.12 2.12 4.24-4.24 1.41 1.41-5.64 5.66z" fill="currentColor"/>
```

Comment:
this is the icon that was mentioned in the updated checklist designs, we're not using it at the moment, but I optimized it already by the time that was decided so I thought I'd just keep it in the library.